enclave_build/src/image_cache.rs

```diff
@@ -289,16 +290,29 @@ pub struct CachePath {}
 
 impl CachePath {
 
-    /// Returns the root folder path of the cache
+    /// Returns the root folder of the cache
+    /// 
+    /// For CACHE_ROOT_FOLDER = XDG_DATA_DIRS
+    // pub fn get_cache_root_folder() -> String {
+    //     let aux = env::var_os(CACHE_ROOT_FOLDER).unwrap();
+    //     let xdg_data_dirs = aux.to_str().unwrap();
+    //     let dirs: Vec<&str> = xdg_data_dirs.split(":").collect();
+
+    //     // For CACHE_ROOT_FOLDER = XDG_DATA_DIRS, choose to take the first directory path
+    //     // PROBLEM - on the EC2 AL2 instance XDG_DATA_DIRS is not set from what I have seen
+    //     // => should have some default root folder for this case
```

Comment:
`$XDG_DATA_HOME defines the base directory relative to which user-specific data files should be stored. If $XDG_DATA_HOME is either not set or empty, a default equal to $HOME/.local/share should be used. `

https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html

Reply:
`$XDG_DATA_DIRS defines the preference-ordered set of base directories to search for data files in addition to the $XDG_DATA_HOME base directory. The directories in $XDG_DATA_DIRS should be seperated with a colon ':'.

If $XDG_DATA_DIRS is either not set or empty, a value equal to /usr/local/share/:/usr/share/ should be used. `

For writing, I would XDG_DATA_HOME or the default, if this is not set. When searching for a container image, the cli should also look into XDG_DATA_DIRS, but I would add this later on.